src/twisted/words/protocols/jabber/client.py

```diff
@@ -314,12 +310,20 @@ def XMPPClientFactory(jid, password):
 
     @param jid: Jabber ID to connect with.
     @type jid: L{jid.JID}
+
     @param password: password to authenticate with.
     @type password: L{unicode}
+
+    @param contextFactory: An object which creates appropriately configured TLS
```

Comment:
Given that this is a new parameter, why not call it `connectionCreator` or `configurationForTLS` rather than `contextFactory`?